### PR TITLE
Make Styling a top-level section

### PR DESCRIPTION
**Pull request by tcmetzger**
As a follow-up to #12467, this PR breaks up the styling chapter into several sub-sections and makes styling its own top-level section.

image

### CODE REVIEW

**Comment by bryevdv** [comment]
I wonder if themes should come last, it sort of ties up all the other styling topics together 

Will also need to move relevant thumbnails and examples for the gallery

**Comment by bryevdv** [comment]
@tcmetzger what do you think about starting with General visual properties as the first section? I also think there's not enough material for render levels to be its own section, but no sure where else to put it.

File: docs/bokeh/make.bat [hunk 1]
```diff
@@ -29,6 +29,7 @@ if "%1" == "help" (
 if "%1" == "clean" (
 	rmdir %BUILDDIR% /s /q
 	del /q /s source\docs\gallery\*
+	del /q /s source\docs\examples\*
```
Comment (tcmetzger) [inline docs/bokeh/make.bat:32]: @bryevdv This is one change I carried over from the Linux Makefile, but I think more updates to make.bat are required. Is that something you could look into? I could also make a separate issue about that!

Comment (bryevdv) [reply docs/bokeh/make.bat:32]: I guess just make an issue, I don't really use windows with any regularity and am not set up to do any dev on windows.

**Comment by bryevdv** [comment]
@tcmetzger can we merge this today before the RC to get the structure in place?

**Comment by tcmetzger** [comment]
@bryevdv sure - we can always make a follow-up PR for the last few details!

**Review by bryevdv — APPROVED** [review]

**Pull request merged**